ikasaneip/ootb/module/scheduler-agent/jar/src/test/resources/application.properties

```diff
@@ -83,3 +83,10 @@ ikasan.exceptions.retry-configs.[0].delayInMillis=5000
 ikasan.exceptions.retry-configs.[0].maxRetries=-1
 
 ikasan.exceptions.excludedClasses[0]=org.ikasan.spec.component.transformation.TransformationException
+
+# Housekeep Log Files Flow
+housekeep.log.files.process.log-folder=./logs
+housekeep.log.files.process.ttl=25
```

Comment:
please change the name of the config value to describe the number value. Is it days for instance?
housekeep.log.files.process.ttl.days